src/frontend/src/components/FormSheet.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import Progress from 'react-progressbar';
+import PollOptionField from './PollOptionField';
+import PollScore from './PollScore';
+import PollInput from './PollInput';
+
+export default class FormSheet extends Component{
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: {},
+            counter: 0,
+            error : false,
+        }
+    }
+
+    setValue(name, value) {
+        this.setState({
+            data: {...this.state.data, [name]: value}  //tworzymy nowy klucz i wartość dla danego elementu
+         })
+    }
+
+    letsSubmit(e){
+        e.preventDefault();
+        this.props.onSubmit(this.state.data)
+    }
+
+    nextQuestion(){
+        if(Object.values(this.state.data)[this.state.counter] !== undefined){
+            if(this.state.counter >= 0 && this.state.counter < this.props.pollData.questions.length){
+                this.setState({
+                    counter: this.state.counter + 1,
+                    error: false
+                })
+            }
+            console.log('next' + this.state.counter)
+        } else {
+            this.setState({
+                error: true
```

Comment:
Instead of setting error to `true`, you should save here error message. Right now you have no way of displaying different validation errors.